widget/embedded/src/components/FilterSelector/FilterSelector.tsx

```diff
@@ -0,0 +1,41 @@
+import type { FilterSelectorPropTypes } from './FilterSelector.type';
+
+import { FilterIcon, Popover } from '@rango-dev/ui';
+import React from 'react';
+
+import { getContainer } from '../../utils/common';
+
+import { Badge, FilterButton, IconContainer } from './FilterSelector.style';
+import { FilterSelectorContent } from './FilterSelectorContent';
+
+export function FilterSelector(props: FilterSelectorPropTypes) {
+  return (
+    <div>
+      <Popover
+        open={props.open}
+        align="end"
+        onOpenChange={props.onOpenChange}
+        container={getContainer()}
+        content={
+          <FilterSelectorContent
+            list={props.list}
+            filterBy={props.filterBy}
+            onClickItem={(id) => {
+              props.onClickItem(id);
```

Comment:
It is better to use the `Destructuring assignment` to maintain code consistency.

```suggestion
              onClickItem(id);
```

Reply:
Fixed